src/main/rules/GCI108/python/GCI108.asciidoc

```diff
@@ -0,0 +1,62 @@
+= Prefer Efficient Classifiers: XGBoost vs RandomForest
+
+Using resource-heavy classifiers like `RandomForestClassifier` for standard classification tasks can result in longer execution times and higher carbon emissions.
+
+`XGBoost` offers a more optimized and eco-friendly alternative with competitive accuracy.
+
+== Metrics Comparison Table
+
+[cols="1,1,1,1", options="header"]
```

Comment:
sorry but I don't understand how you do your tests ... could you give us more information on your testing context, please ?